core/cmake/BareosFindAllLibraries.cmake

```diff
@@ -216,4 +216,18 @@ if(ENABLE_JANSSON)
   find_package(Jansson)
 endif()
 
+if(USE_SYSTEM_CLI11)
+  find_package(CLI11 "2.2.0" CONFIG REQUIRED)
+  message(STATUS "Using system CLI11 ${CLI11_VERSION}")
+endif()
+  find_package(fmt "6.2.1" CONFIG REQUIRED)
+if(USE_SYSTEM_FMT)
+  message(STATUS "Using system fmt ${fmt_VERSION}")
+endif()
+if(USE_SYSTEM_XXHASH)
+  find_package(PkgConfig REQUIRED)
+  pkg_search_module(XXHASH REQUIRED IMPORTED_TARGET xxhash>=0.8.1 )
```

Comment:
That doesn't find the installed xxhash on my RHEL 8 or RHEL 9 machines.

Reply:
Very interesting on my systems it will work.

Reply:
Did you build your own xxhash packages? Did you patch the pkgconfig files for that?
Because on my machine it is "libxxhash" and not "xxhash".
```
$ lsb_release -d
Description:	Red Hat Enterprise Linux release 8.8 (Ootpa)
$ rpm -q xxhash-devel
xxhash-devel-0.8.1-3.el8.x86_64
$ pkg-config --list-all | grep xxhash
libxxhash                      xxhash - extremely fast hash algorithm
```